pep-0007.txt

```diff
@@ -124,11 +124,13 @@ Code lay-out
                    type->tp_name);
 
 * When you break a long expression at a binary operator, the
-  operator goes at the end of the previous line, e.g.::
+  operator goes at the end of the previous line, and braces should be
```

Comment:
That actually looks a bit weird to me, though I can understand the rationale.  I'd personally prefer a rule that states the open brace not to be hanging in all multiline conditional cases.